### ISSUE TITLE
LCD2USB doesn't work unless started with sudo

### ISSUE THREAD

**Issue by pattdepanick** [pattdepanick/vlub#27]
service crashing at startup : fix by using sudo ./roselcd.py

**Comment by pattdepanick** [comment]
vlub installer should create a user or a group which will own the devices and then launch the process as the user  / group created. This would prevent from running root as it is today.

**Comment by Didier-Lalli** [comment]
Device is opened with RW access. volumio user has only read access.

```bash
root@volumio:~# ls -al /dev/bus/usb/001/
total 0
drwxr-xr-x 2 root root    140 Jan  1  1970 .
drwxr-xr-x 3 root root     60 Jan  1  1970 ..
crw-rw-r-- 1 root root 189, 0 Feb 11 18:06 001
crw-rw-r-- 1 root root 189, 1 Feb 11 18:06 002
crw-rw-r-- 1 root root 189, 2 Feb 11 18:06 003
crw-rw-r-- 1 root root 189, 3 Feb 11 18:06 004
crw-rw-rw- 1 root root 189, 4 Feb 11 18:12 005
```

Solution would be to use lsusb to detect device and retrieve the device ID and change the access right at install time.

```
volumio@volumio:~/vlub$ lsusb
Bus 001 Device 005: ID 0403:c630 Future Technology Devices International, Ltd lcd2usb interface
Bus 001 Device 004: ID 2341:814e Arduino SA 
Bus 001 Device 003: ID 0424:ec00 Standard Microsystems Corp. SMSC9512/9514 Fast Ethernet Adapter
Bus 001 Device 002: ID 0424:9514 Standard Microsystems Corp. 
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
```

**Comment by Didier-Lalli** [comment]
Try to include in udev rules:

SUBSYSTEM=="usb", ATTRS{product}=="LCD2USB Interface", GROUP="volumio"

**Comment by pattdepanick** [comment]
fixed with Merge pull request #28 from bcornec/master
